src/util/logging.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+import (
+	"fmt"
+	"github.com/logrusorgru/aurora"
+	"log"
+)
+
+func LogError(msg interface{}) {
+	log.Println(aurora.Red("[ERROR] " + fmt.Sprintf("%v", msg)).Bold()) // Sprintf for nil printing
```

Comment:
I'd rather keep the vargs offered by the exsting `printf` and `println` options. It's nice being able to use format strings when needed or pass an arbitrary number of arbitrary `interface{}`s into the println method and have them automatically deserialised into text.